### PR TITLE
Side effect exports 2

### PR DESCRIPTION
**Pull request by jridgewell**
Removes a few more side-effects of importing that can't be DCE'd.

### CODE REVIEW

File: src/error.js [hunk 1]
```diff
@@ -21,10 +21,19 @@ import {USER_ERROR_SENTINEL, isUserErrorMessage} from './log';
...
 
+/**
+ * A wrapper around our exponentialBackoff, to lazy initialize it to avoid an
+ * un-DCE'able side-effect.
+ * @param {function()} work the function to execute after backoff
+ * @return {number} the setTimeout id
+ */
+let globalExponentialBackoff = function(work) {
+  // Set globalExponentialBackoff as the lazy-created function. JS Vooodoooo.
+  globalExponentialBackoff = exponentialBackoff(1.5)
```
Comment (aghassemi) [inline src/error.js]: not to take away from the awesomness of this vodoo, but add `;` :)

File: src/experiments.js [hunk 2]
```diff
@@ -36,8 +36,16 @@ const COOKIE_EXPIRATION_INTERVAL = COOKIE_MAX_AGE_DAYS * 24 * 60 * 60 * 1000;
 /** @const {string} */
 const CANARY_EXPERIMENT_ID = 'dev-channel';
 
-/** @const {!Object<string, boolean>} */
-const EXPERIMENT_TOGGLES = Object.create(null);
+/** @type {Object<string, boolean>} */
+let toggles;
```
Comment (aghassemi) [inline src/experiments.js]: I am a tiny bit worried about this and similar ones ( `templateTagSupported` ) being accessed directly. We should hide them more. Either by scary names ( `uninitializedFoo` , `fooCache` ) or hide them all inside another object

File: src/experiments.js [hunk 3]
```diff
@@ -36,8 +36,16 @@ const COOKIE_EXPIRATION_INTERVAL = COOKIE_MAX_AGE_DAYS * 24 * 60 * 60 * 1000;
...
+
+/**
+ * A wrapper to avoid a static side-effect.
+ * @return {!Object<string, boolean>}
+ */
+function experimentsToggles() {
```
Comment (aghassemi) [inline src/experiments.js]: nit: experimentsToggles or experimentToggles?

**Comment by aghassemi** [comment]
Minor comments
LGTM

File: src/experiments.js (hunk 2 above)
Comment (jridgewell) [inline src/experiments.js]: Fixed.

File: src/error.js (hunk 1 above)
Comment (jridgewell) [inline src/error.js]: Fixed.

File: src/experiments.js (hunk 3 above)
Comment (jridgewell) [inline src/experiments.js]: Fixed.

**Pull request merged**